tests/handlers/records-write.spec.ts

```diff
@@ -3020,7 +3020,7 @@ export function testRecordsWriteHandler(): void {
             recipient    : alice.did
           });
           const imageReply = await dwn.processMessage(alice.did, imageRecordsWrite.message, { dataStream: imageRecordsWrite.dataStream });
-          expect(imageReply.status.code).to.equal(202); // allows write but is not readable or queryable
+          expect(imageReply.status.code).to.equal(204); // allows write but is not readable or queryable
```

Comment:
This seems suspect, but I saw that https://github.com/TBD54566975/dwn-sdk-js/blob/c6d8526e4aa83336b08f646d906c14c435b13311/tests/utils/test-data-generator.ts#L416 is not kicking in

Reply:
Fair observation, let me update the test!

Reply:
Updated.